docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
     volumes:
       # we need the notebooks mount to run the tests (since we don't copy them in)
       - ./notebooks:/home/jovyan/data-science-dags/notebooks
+      - ./notebooks_tmp:/tmp
```

Comment:
Have you noticed any difference in memory consumption? Maybe `/tmp` is a good path for a host temporary directory?

I guess because you are not using the `jupyter` docker service, this wouldn't affect you but maybe doing it for airflow would? In the end, Kubernetes is probably the only place where that is important unless you are running out of disk space too.

Reply:
I have added the volume to dask-worker, when I try to test locally I am not able see a huge difference but hopefully on prod we can.